### ISSUE TITLE
EndOfRunMonitor stops sending runs to the queue

### ISSUE THREAD

**Issue by jsws** [ISISScientificComputing/autoreduce#103]
After a certain period of time, (I'm assuming time but it very well could be after a certain number of events) EndOfRunMonitor will stop sending runs to the queue.
It gives up on one instrument at a time, given enough time it wouldn't send runs for any of the instruments.
It works again fine once it's restarted. 

 I believe I've tried to debug this problem before and found that it stops receiving the `modified FileSystemEventHandler` from `last_run.txt`.
In other words, this method is no longer called when a `last_run.txt` is updated on the ISIS archive.
https://github.com/ISISScientificComputing/autoreduce/blob/36097bf5bbf18712ff5ad3f780af7803ec13b206/EndOfRunMonitor/ISISendOfRunMonitor.py#L117

One suggested cause is the FileSystemEventHandler is left open for too long on the ISIS network drive and something times out.

**Comment by ElliotAOram** [comment]
system for detecting and resolving this issue via restarting the service will be available in #205
